modules/nf-core/parabricks/fq2bam/main.nf

```diff
@@ -0,0 +1,73 @@
+process PARABRICKS_FQ2BAM {
+    tag "$meta.id"
+    label 'process_high'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used with Parabricks at the moment. Please use docker or singularity."
+    }
+
+    container "nvcr.io/nvidia/clara/clara-parabricks:4.0.0-1"
+
+    input:
+    tuple val(meta), path(reads), path(interval_file)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(index)
+    path known_sites
+    val markdups
+    val qc_metrics
+
+    output:
+    tuple val(meta), path("*.bam"), emit: bam
+    tuple val(meta), path("*.bai"), emit: bai
+    path "versions.yml", emit: versions
+    path "qc_metrics", optional:true, emit: qc_metrics
+    path("*.table"), optional:true, emit: bqsr_table
+    path("*-duplicate-metrics.txt"), optional:true, emit: duplicate_metrics
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def in_fq_command = meta.single_end ? "--in-se-fq $reads" : "--in-fq $reads"
+    def mark_duplicates_command = markdups ? "" : "--no-markdups"
+    def mark_duplicates_output = markdups ? "--out-duplicate-metrics ${prefix}-duplicate-metrics.txt" : ""
+    def qc_metrics = qc_metrics ? "--out-qc-metrics-dir qc_metrics" : ""
+    def known_sites_command = known_sites ? known_sites.collect{"--knownSites $it"}.join(' ') : ""
+    def known_sites_output = known_sites ? "--out-recal-file ${prefix}.table" : ""
+    def interval_file_command = interval_file ? interval_file.collect{"--interval-file $it"}.join(' ') : ""
+    """
+
+    INDEX=`find -L ./ -name "*.amb" | sed 's/.amb//'`
```

Comment:
```suggestion
    INDEX=`find -L ./ -name "*.amb" | sed 's/\\.amb\$//'`
```

Reply:
Good catch, I see you also got that fixed in `bwa`!